.github/workflows/prettier.yml

```diff
@@ -1,25 +1,48 @@
 name: Format
 on:
   push:
-    paths:
-      - "data/**"
+    branches:
+      - main
```

Comment:
Would it make sense to have both of these?

Prettier only changes the files in the data directory,
so only run it on changes to the data directory pushed to main.

Reply:
Unfortunately, adding several triggers to a workflow chains them as an `OR` expression, not an `AND` expression. Therefore, we can't use this approach, because then we would trigger the `prettier.yml` workflow for pushes to arbitrary branches.

Reply:
I use the following on one of my repos:
```yml
  push:
    branches:
      - main
    paths-ignore:
      - ".github/**"
 ```

Is the reason this works because it is `paths-ignore`?

Reply:
You're right, this does work. `branches` and `paths`/`paths-ignore` in combination actually get interpreted as an `AND` expression...

I'd rather leave it like this though, because now the `release.yml` workflow depends on `prettier.yml` so we want `prettier.yml` to run for every push to `main` in order to check whether we can cut a new release afterwards.

Reply:
OK